### DISCUSSION TITLE
Palisades

### DISCUSSION THREAD

**Discussion by thomasherena** [Keriew/augustus#529] [Q&A] [answered]
While going through the asset files, I saw that there was a palisade file. Is this building implemented yet?

**Comment by MSTVD** [comment, marked as answer]
Yes, in the unstable version.

**Comment by Mephisto-Fibs** [comment]
I think the palisades should be bigger; Maybe through thicker and / or taller trunks. Even the newly added garden wall feels more massive than the palisade fence.

I suggest that it could behave like the vanilla walls; which when you put it next to each other connect and becomes thicker and more massive. The guards from the new watchtowers could patrol ontop the palisades instead of the streets. The watchtowers could be built into the palisades in a similar way as the vanilla wall towers.  

This way the pallisades in a way model the Roman Limes.
